File: primer_trimestre/poo/php/ejercicios_php/third/libreria/firstLaw.php
<?php
require_once 'NewtonLaw.php';
class FirstLaw extends NewtonLaw {
    public function __construct(float $mass, float $acceleration, float $force) {
        parent::__construct($mass, $acceleration, $force);
    }

    public function calculate(): string {
        $force = $this->getForce();
        if ($force == 0) {
            return "force at rest";
        } else {
            return "force in motion";
        }
    }
}

File: primer_trimestre/poo/java/proyectos_java/ejercicio_3_java/src/firstLaw.java
public class firstLaw extends NewtonLaw {
    public firstLaw(double mass, double acceleration, double force) {
        super(mass, acceleration, force);
    }
    @Override
    public String calculate() {
        double force = getForce();
        if(force==0){
            return ("force at rest" );
        } 
        else{
            return ("force in motion" );
        }
    }

}
